grid.cpp

```diff
@@ -491,11 +485,12 @@ void balanceLoad(dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geometry>& mpiGrid, S
    // tell other processes which velocity blocks exist in remote spatial cells
    phiprof::Timer balanceLoadTimer {"Balancing load", {"Load balance"}};
 
+   // memory deallocation for dAMR. Do we want this got GPUs as well?
    phiprof::Timer deallocTimer {"deallocate boundary data"};
    //deallocate blocks in remote cells to decrease memory load
```

Comment:
Both comments can be merged – does the first one mean it's not doing this on GPUs atm?